ironfish/src/network/peerNetwork.ts

```diff
@@ -317,16 +323,6 @@ export class PeerNetwork {
     this.updateIsReady()
   }
 
-  /**
-   * Send the message to all connected peers with the expectation that they
-   * will forward it to their other peers. The goal is for everyone to
-   * receive the message.
-   */
-  private gossip(message: GossipNetworkMessage): void {
```

Comment:
This is trivial, just moving this logic to the above so we can mark peers as having seen transaction hashes